### DISCUSSION TITLE
It's been about 5 days since I've been approved for benefits, where's the email???

### DISCUSSION THREAD

**Discussion by ultraplay312** [community/community#69133] [GitHub Education]
### Hi everyone,

.

**Comment by samus-aran** [comment]
Hi @ultraplay312  👋

We're sorry you're experiencing this. Can we please ask you to try now and re-apply. We've raised this experience with our internal team but we hope this helps you proceed to re-verify your academic status. 👍

**Reply by ultraplay312** [reply]
That wasn't the problem. I'm verfied it said; It's just taking awhile and I'm just asking what the avg wait time is. I got approved on 9/30/23

**Reply by ultraplay312** [reply]
ill do it just in case though

**Reply by ultraplay312** [reply]
ok so I did need to re-apply but I'm not sure if you did that or if it was already like that. I'll post an update if I get my benefits soon

**Comment by ultraplay312** [comment]
Still nothing

**Reply by samus-aran** [reply]
Hi @ultraplay312 👋 

We revoked your first application (the one that was stuck) which then requires you to re-apply. We can see you have since re-applied so now you should receive your academic benefits within a time frame. You have access to Global Campus right away. Your educational benefits will arrive within 72 hours of this verification. **However, during busy periods like the start of a new semester, it may take a bit longer.** If you haven't received your academic benefits within 5 days please re-reply to this thread 👍 But let us know if you do receive them and we can close this 🙂

**Comment by mantasu** [comment]
I think I had a similar issue (had to wait until "Approved" tag became purple). Actually, if you visit the [ticket](https://support.github.com/request/education) page, it says:

> Applications may take several weeks to be approved

It seems the staff has re-raised the issue, thus if you haven't received the benefits yet, you just need to wait for a bit more but you'll get access soon!
